e2e/main_test.go

```diff
@@ -5,6 +5,9 @@ import (
 	"testing"
 )
 
+// only difference between integrationTests and testnetIntegrationTests is
+// that integration tests are run against memstore whereas testnetintegration tests
+// are run against eigenda backend talking to testnet disperser
```

Comment:
this does change the conditional behavior of some of the tests - e.g, in `TestProxyServerCaching` we only assert to read metrics with EigenDA when referencing memstore since we don't profile the `eigenDAClient` interactions

Reply:
[7ccf0c1](https://github.com/Layr-Labs/eigenda-proxy/pull/177/commits/7ccf0c191981d6fdd52daf74859800067d7b7a97)